Tools/Scripts/filter-jsc-tests.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Apple Inc. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import argparse
+import sys
+
+
+def parser():
+    parser = argparse.ArgumentParser(description='filter test output')
+    parser.add_argument(
+        '--log-output',
+        dest='output_dir',
+        help='output directory for unfiltered log',
+        default='logs.txt'
+    )
+
+    return parser.parse_args()
+
+
+def filter_jsc_test(log_file_name):
+    log_file = os.path.abspath(f'{log_file_name}')
+    with open(log_file, "w") as f:
+        for line in sys.stdin:
+            if not line.startswith('Running') and not line.startswith('Skipping'):
```

Comment:
Please check how much is the number of output lines after that filtering. For example: for this run with 500k+ lines, how much is the number of lines of output https://build.webkit.org/#/builders/64/builds/225239

Reply:
Oh that's a good point. I changed the filtering so it only prints lines after a line starts with `**` or `Results`, which indicates all tests have finished and results are being printed.